.github/workflows/docker-release.yml

```diff
@@ -31,10 +32,22 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Determine Tag
+        id: tag
+        run: |
+          if [ "${{ github.ref_type }}" = "tag" ] && [ "${{ github.ref }}" != "refs/heads/staging" ]; then
+            echo "tag_name=${{ github.ref_name }}" >> $GITHUB_OUTPUT
+          else
+            echo "tag_name=staging" >> $GITHUB_OUTPUT
```

Comment:
Does this mean all staging builds will be tagged as staging without version number e.g. `mcsystems/news-search-api:staging` ? How would we be able to tell one build from the next? I think we should at least include commit hash, `${{ github.sha }}`, as (part of) `tag_name`.

Of course, there is also [Docker Metadata Action](https://github.com/marketplace/actions/docker-metadata-action).

Reply:
We could us "beta" version tags (ie; suffixed with b1 etc) for staging